tests/test_using.py

```diff
@@ -1581,7 +1599,7 @@ class Meta:
             book__author__country = factory.LazyAttribute(lambda o: 'FR')
 
         chapter = ChapterFactory()
-        self.assertEquals('FR', chapter.book.author.country)
+        self.assertEqual('FR', chapter.book.author.country)
```

Comment:
Was getting a deprecation warning on this:

```
test_using.py:1601: DeprecationWarning: Please use assertEqual instead.
```

Reply:
Thanks for the fix. IMO, this should be pulled out in a separate PR since it's not related to #504.

Reply:
This change has been handled in a separate PR: https://github.com/FactoryBoy/factory_boy/pull/518.